acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/ACSAcceptanceIT.java

```diff
@@ -79,74 +72,51 @@
     private PolicyHelper policyHelper;
 
     @Autowired
-    private ZacTestUtil zacTestUtil;
-
-    @Autowired
-    private Environment env;
-
-    @Autowired
     private ACSRestTemplateFactory acsRestTemplateFactory;
 
-    @Value("${ZONE1_NAME:testzone1}")
-    private String acsZone1Name;
-
-    @Value("${ZONE2_NAME:testzone2}")
-    private String acsZone2Name;
+    @Autowired(required = false)
+    private SetupUtility zacHelperUtil;
 
-    @Value("${ZONE3_NAME:testzone3}")
-    private String acsZone3Name;
-
-    @Value("${ACS_UAA_URL}")
+    @Value("${ZONE1_UAA_URL}")
     private String uaaUrl;
 
     private OAuth2RestTemplate acsAdminRestTemplate;
-    private boolean registerWithZac;
+    private HttpHeaders headersWithZoneSubdomain = new HttpHeaders();
+    private NewUaaTestUtil uaaTestUtil;
+    private String acsZoneName;
 
     @BeforeClass
-    public void setup() throws JsonParseException, JsonMappingException, IOException {
-        if (Arrays.asList(this.env.getActiveProfiles()).contains("public")) {
-            setupPublicACS();
-        } else {
-            setupPredixACS();
-        }
-        this.headersWithZoneSubdomain = new HttpHeaders();
-        this.headersWithZoneSubdomain.set("Predix-Zone-Id", this.testZoneSubdomain);
-    }
+    public void setup() throws IOException {
```

Comment:
To summarize, common setup for public and predix profiles should perform the following:
- Derive test zone name from a test class name and randomize with a timestamp.  
- Create ACS admin client in UAA using pre-existing UAA admin client.  Client name and secret are derived from test zone name.
- Create a test zone in ACS using ACS admin client.  Upon zone creation, zone could be optionally registered with ZAC if zacRestTemplate bean is present.